actionpack/test/controller/render_json_test.rb

```diff
@@ -63,6 +63,13 @@ def render_json_with_extra_options
     def render_json_without_options
       render json: JsonRenderable.new
     end
+
+    def render_plain_text_for_xml_render_to_string
+      @inline_template = render_to_string(inline: <<-XML, formats: [:xml])
+        <language>Ruby</language>
+      XML
+      render plain: "Hello"
```

Comment:
The original report seemed to be about `render :nothing` (which I believe is gone), should we also test `head :ok` with a `render_to_string`?